documentation/modules/configuring/con-mm2-recovery.adoc

```diff
@@ -0,0 +1,41 @@
+// This module is included in:
+//
+// assembly-config.adoc
+
+[id="con-mm2-recovery-{context}"]
+= Disaster recovery in an active/passive configuration
+
+[role="_abstract"]
+MirrorMaker 2 can be configured for active/passive disaster recovery.  
+To support this, the Kafka cluster should also be monitored for health and performance to detect issues that require failover promptly.
+
+If failover occurs, which can be automated, operations switch from the active cluster to the passive cluster when the active cluster becomes unavailable.
+The original active cluster is typically considered permanently lost.
+The passive cluster is promoted to active status, taking over as the source for all application traffic.
+In this state, MirrorMaker 2 stops replicating data from the original active cluster because it is no longer functional.
```

Comment:
This makes it sound like MirrorMaker 2 stops automatically. If the original active cluster came back online Mirror Maker 2 would start replicating again. Should we add a step below to remove the Mirror Maker 2 configuration for the original active -> passive mirror?